### ISSUE TITLE
Conflict with tab focus addons

### ISSUE THREAD

**Issue by r-a-y** [rNeomy/auto-tab-discard#14]
Great addon for Firefox Quantum!

One thing I'm finding is a conflict with using tab focus addons like "Tab Deque" or "Tab Open/Close Control".

Tab Deque is an addon that helps to give focus to the last active, selected tab when you close a tab.  When this happens, Tab Deque looks like it gives focus to an adjacent tab before allowing to focus on the last active tab.

For Auto Tab Discard, if the adjacent tab was discarded since Tab Deque will give focus, the tab will now be undiscarded before focusing on the last active tab.

I know this probably isn't a problem with Auto Tab Discard, but thought I would note this.  Appears to be a problem with all tab focus addons, so I'm guessing this is a Firefox tab API limitation at the moment.

**Comment by rNeomy** [comment]
Right, I don't see how we can fix this. This extension uses the built-in discarding API and by design, it gets undiscarded when the tab is focused. Let's keep this open for ideas.

**Comment by rayman89** [comment]
You could ask the other addon guys to include an option to select closest active tab. That was possible before but idk if it's possible with webext.

**Comment by Jormangeud** [comment]
Thanks for explanation rNeomy.
This undiscarding on tab focus makes it extremely painful to change/browse tabs with the keyboard.
